### ISSUE TITLE
[REFACTOR] ATOMS adjust Storefront UI design system

### ISSUE THREAD

**Issue by pspaczek** [DivanteLtd/storefront-ui#804]
rewrite atoms CSS for a new design system http://bit.ly/storefront-ui-design-system

Workflow:
1. create a branch from `refactor/design-system`
2. create PR with your changes

Notes: 
1. **don't** touch components props API
2. note on PR all changes in component CSS modifiers API
3. in components styles you should use **always** css variables, don't use hardcode values for spacers, colour's, font's etc.
4. spacers should be align to 8px grid like on Google Matrial
5. refactor workflow
- move css to outide file
- set style for base component
a. set font
b. set spacers
c. set color's
d. set others property
- create modifiers
- test it 🙏
- push it
6. we shouldn't use different element font size in component, it should be the same
``` scss
// *don't
.sf-button{
font-size: --body-font-size-regular;
  @include for-desktop {
    font-size: --body-font-size-extra-big;
  }
}
```
7.  scss filer structure
``` scss
@import "../../helpers";
.sf-button{
  width: 
  height:
  padding:
  @include for-desktop {
  --button-padding:
  }
  &--modifier {
    --button-padding:
   }
}
```
Scope of work:
*please assigne you to component 
- [x] Arrow - @aniamusial 
- [x] Badge - @aniamusial 
- [x] Breadcrumbs - @justyna-13 
- [x] Bullets - @justyna-13 
- [x] Button - @aniamusial 
- [x] Checkbox @martaradziszewska 
- [x] Chevron @AdamPawlinski 
- [x] CircleIcon @martaradziszewska 
- [x] Color @martaradziszewska 
- [x] Divider
- [x] Heading @martaradziszewska 
- [x] Icon @pspaczek
- [x] Image @pspaczek
- [x] Input @martaradziszewska 
- [x] Loader @justyna-13 
- [x] Overlay @martaradziszewska 
- [x] Price @aniamusial 
- [x] Property @AdamPawlinski 
- [x] QuantitySelector @AdamPawlinski 
- [ ] Rating @justyna-13 

in refactor/design-system:
- [x] Arrow - @aniamusial 
- [x] Badge - @aniamusial 
- [x] Breadcrumbs - @justyna-13 
- [x] Bullets - @justyna-13 
- [x] Button - @aniamusial 
- [x] Checkbo  @martaradziszewska 
- [x] Chevron @AdamPawlinski 
- [x] CircleIcon @martaradziszewska 
- [x] Color @martaradziszewska 
- [x] Divider
- [x] Heading @martaradziszewska 
- [x] Icon @pspaczek
- [x] Image @pspaczek
- [x] Input @martaradziszewska 
- [x] Loader @justyna-13 
- [x] Overlay @martaradziszewska 
- [x] Price @aniamusial 
- [x] Property @AdamPawlinski 
- [x] QuantitySelector @AdamPawlinski 
- [x] Rating @justyna-13

**Comment by AdamPawlinski** [comment]
Wezmę Chevron

**Comment by justyna-13** [comment]
Wezmę jeszcze Loader

**Comment by pspaczek** [comment]
merged all atoms to #871
